src/deploy.jl

```diff
@@ -102,7 +102,7 @@ function deploy(
     if versioned
         ## Find the current versions.
         dir = joinpath(filter(!isempty, [dir, name])...)
-        versions = sort!(filter(s->!startswith(s, '.'), readdir(dir)); rev=true)
+        versions = sort!(filter(s->!startswith(s, '.') && isdir(s), readdir(dir)); rev=true)
```

Comment:
`isdir(s)` should probably be `isdir(dir, s)` otherwise it checking from the wrong directory I think and won't pick up the right folders.

Reply:
Done